misc/config_tools/board_inspector/legacy/board_parser.py

```diff
@@ -46,6 +46,16 @@ def check_env():
     if os.path.exists(PY_CACHE):
         shutil.rmtree(PY_CACHE)
 
+    # check msr driver
+    res = subprocess.Popen("lsmod | grep -i ^msr",
+                            shell=True, stdout=subprocess.PIPE,
+                            stderr=subprocess.PIPE, close_fds=True)
+    line = res.stdout.readline().decode('ascii')
```

Comment:
The key here is the make sure `/dev/cpu/0/msr` does exist. With kernels that have `CONFIG_X86_MSR` being `y`, there is no module named `msr` but the file is still accessible.

Thus I would recommend to check the existence of `/dev/cpu/X/msr` where `X` is any directory under `/dev/cpu` (you may use `os.listdir` and `os.path.isdir` to get the list of directories there).

Reply:
fixed.